Formula/arrayfire.rb

```diff
@@ -25,20 +25,10 @@ class Arrayfire < Formula
   fails_with gcc: "5"
 
   def install
-    # Fix for: `ArrayFire couldn't locate any backends.`
-    # We cannot use `CMAKE_INSTALL_RPATH` since upstream override this:
-    #   https://github.com/arrayfire/arrayfire/blob/590267d2/CMakeModules/InternalUtils.cmake#L181
-    linker_flags = [
-      rpath(source: lib, target: Formula["fftw"].opt_lib),
-      rpath(source: lib, target: Formula["openblas"].opt_lib),
-      rpath(source: lib, target: HOMEBREW_PREFIX/"lib"),
-    ]
-    linker_flags.map! { |path| "-Wl,-rpath,#{path}" }
-
     system "cmake", "-S", ".", "-B", "build",
                     "-DAF_BUILD_CUDA=OFF",
                     "-DAF_COMPUTE_LIBRARY=FFTW/LAPACK/BLAS",
-                    "-DCMAKE_SHARED_LINKER_FLAGS=#{linker_flags.join(" ")}",
+                    "-DCMAKE_INSTALL_RPATH=#{rpath}",
```

Comment:
If you want to use `CMAKE_INSTALL_RPATH` then you need to use the original array above instead of just `rpath`.

Reply:
Upstream no longer overrides `CMAKE_INSTALL_RPATH`, so I was attempting to remove the workaround. The build fails whether this change is made or not. Any guidance here would be appreciated.

Reply:
Yes, we can use `CMAKE_INSTALL_RPATH` instead of `CMAKE_SHARED_LINKER_FLAGS`, but the contents of the `CMAKE_INSTALL_RPATH` argument must be equivalent to the content of `CMAKE_SHARED_LINKER_FLAGS`.

So it should look something like this:
```diff
diff --git a/Formula/arrayfire.rb b/Formula/arrayfire.rb
index 63580e231fa..35377436e13 100644
--- a/Formula/arrayfire.rb
+++ b/Formula/arrayfire.rb
@@ -26,19 +26,16 @@ class Arrayfire < Formula
 
   def install
     # Fix for: `ArrayFire couldn't locate any backends.`
-    # We cannot use `CMAKE_INSTALL_RPATH` since upstream override this:
-    #   https://github.com/arrayfire/arrayfire/blob/590267d2/CMakeModules/InternalUtils.cmake#L181
-    linker_flags = [
+    rpaths = [
       rpath(source: lib, target: Formula["fftw"].opt_lib),
       rpath(source: lib, target: Formula["openblas"].opt_lib),
       rpath(source: lib, target: HOMEBREW_PREFIX/"lib"),
     ]
-    linker_flags.map! { |path| "-Wl,-rpath,#{path}" }
 
     system "cmake", "-S", ".", "-B", "build",
                     "-DAF_BUILD_CUDA=OFF",
                     "-DAF_COMPUTE_LIBRARY=FFTW/LAPACK/BLAS",
-                    "-DCMAKE_SHARED_LINKER_FLAGS=#{linker_flags.join(" ")}",
+                    "-DCMAKE_INSTALL_RPATH=#{rpaths.join(";")}",
                     *std_cmake_args
     system "cmake", "--build", "build"
     system "cmake", "--install", "build"
```

Reply:
That makes sense. Thank you!